src/components/nav.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import styled from "styled-components"
+import colors from "../assets/colors"
+import Icon from "./icon"
+
+const Nav = styled.div`
+  border-bottom: 1px solid ${ colors.grayRegular };
+  display: flex;
+`
+
+const NavIcon = styled.div`
+  border-right: 1px solid ${ colors.grayRegular };
+  display: inline-block;
+  font-size: 32px;
+  padding: 10px 5px 5px 5px;
+  margin-right: 10px;
+`
```

Comment:
@l2succes @xtina-starr we just had a conversation offline, but since we are using `display: flex` already in `Nav`, I'm throwing out the diff I just made to kill magic numbers...

```diff
+const NavHeight = 54
+
 const Nav = styled.div`
   border-bottom: 1px solid ${ colors.grayRegular };
   display: flex;
+  box-sizing: border-box;
+  height: ${NavHeight}px;
 `

 const NavIcon = styled.div`
   border-right: 1px solid ${ colors.grayRegular };
-  display: inline-block;
+  display: flex;
   font-size: 32px;
-  padding: 10px 5px 5px 5px;
+  align-self: stretch;
+  align-items: center;
+  justify-content: center;
   margin-right: 10px;
+  box-sizing: border-box;
+  width: ${NavHeight}px;
 `
```